web/docs/use-vast/ingest/README.md

```diff
@@ -1,31 +1,26 @@
 # Ingest
 
-*Ingesting* data refers to the import pipeline that covers the following steps:
+Sending data to VAST (aka *ingesting*) involves spinning up a VAST client node
```

Comment:
What's a "client node"? You've written this multiple times, but the term isn't explained anywhere.

Reply:
Well observed. I'm going to describe this in the upcoming Run section, in future commits within this PR.

Reply:
I clarified occurrences of "node".